taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/infrastructure/media/writer/AutocompletionResultWriter.java

```diff
@@ -0,0 +1,37 @@
+package com.digirati.taxman.rest.server.infrastructure.media.writer;
+
+import com.digirati.taxman.rest.server.taxonomy.autocomplete.AutocompletionResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import javax.ws.rs.ext.Provider;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Provider
+public class AutocompletionResultWriter implements MessageBodyWriter<AutocompletionResult> {
```

Comment:
Why was this needed?

Reply:
This was needed to serialise the `AutocompletionResult` as JSON. Without it there's a runtime error complaining that this doesn't exist

Reply:
Didn't this work? https://quarkus.io/guides/rest-json-guide

Reply:
No. Presumably it's because there's already a `MessageBodyWriter` for `RdfModel` so it's trying to use that API for every endpoint

Reply:
We can get rid of this anyway when we send RDF data back to the client.